.github/workflows/e2e_core.yml

```diff
@@ -184,8 +173,14 @@ jobs:
         echo "LIT_XFAIL=${{inputs.xfail}}" >> $GITHUB_ENV
         echo "LIT_FILTER_OUT=${{inputs.filter_out}}" >> $GITHUB_ENV
 
-    - name: Run e2e tests
+    - name: Run e2e tests (on schedule)
+      if: inputs.trigger == 'schedule'
+      run: ninja -C build-e2e check-sycl-e2e
+
+    - name: Run e2e tests (on PR; allow failures)
+      if: inputs.trigger != 'schedule'
       id: tests
+      continue-on-error: true
```

Comment:
should we make just this step "continue-on-error" or the whole job...?

Reply:
The whole job probably. There might be scenarios (e.g., API changes in UR) where SYCL won't even compile.

Reply:
Done.